crates/holochain/src/core/ribosome/host_fn/get_agent_key_lineage.rs

```diff
@@ -0,0 +1,67 @@
+use crate::core::ribosome::CallContext;
+use crate::core::ribosome::RibosomeError;
+use crate::core::ribosome::RibosomeT;
+use holochain_types::prelude::*;
+use holochain_wasmer_host::prelude::*;
+use std::sync::Arc;
+use wasmer::RuntimeError;
+
+#[allow(clippy::extra_unused_lifetimes)]
```

Comment:
Is this allow actually needed or can the lifetime be removed? I think this is a hangover from an earlier Rust version

Reply:
No, doesn't seem to be needed.